styles/home.css

```diff
@@ -0,0 +1,150 @@
+.homepage_main_container {
+  width: 100%;
+}
+
+.homepage_banner_container {
+  height: 85vh;
+  background: url(https://cdn.shopify.com/s/files/1/1485/6894/files/home5_bg1.jpg?v=1514625150);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 1.5rem;
+}
+
+.banner_content_container {
+  text-align: center;
+  padding-top: 6rem;
+}
+
+._title {
```

Comment:
any specific reason for this class name?

Reply:
Yes actually made those class, it is not getting used right now , but I have made those to use in future